config/dirs_windows.go

```diff
@@ -143,6 +154,10 @@ func driverInfoFromKey(k registry.Key, driverName string) (di DriverInfo, err er
 	di.Driver.Shared.defaultPath = keyMust(dkey, "driver")
 	di.Driver.Entrypoint = keyOptional(dkey, "entrypoint")
 
+	// For drivers in the registry, set FilePath to the registry key instead
+	// of the filesystem path since that's technically where the driver exists.
+	di.FilePath = fmt.Sprintf("%s\\%s", lvl.rootKeyString(), regKeyADBC)
```

Comment:
Since we have k and dkey, can't we get the full key names from those instead of needing the new function?

Reply:
I looked again and I don't see how. It looks like keys are comparable so we can at least simplify it a bit. I'd be happy if you took a look at the package too just in case I'm missing something.